server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/futurocoin/FuturocoinExtension.java

```diff
@@ -0,0 +1,133 @@
+/*************************************************************************************
+ * Copyright (C) 2014-2016 GENERAL BYTES s.r.o. All rights reserved.
+ *
+ * This software may be distributed and modified under the terms of the GNU
+ * General Public License version 2 (GPL2) as published by the Free Software
+ * Foundation and appearing in the file GPL2.TXT included in the packaging of
+ * this file. Please note that GPL2 Section 2[b] requires that all works based
+ * on this software must also be made publicly available under the terms of
+ * the GPL2 ("Copyleft").
+ *
+ * Contact information
+ * -------------------
+ *
+ * GENERAL BYTES s.r.o.
+ * Web      :  http://www.generalbytes.com
+ *
+ ************************************************************************************/
+package com.generalbytes.batm.server.extensions.extra.futurocoin;
+
+import com.generalbytes.batm.server.extensions.*;
+import com.generalbytes.batm.server.extensions.extra.futurocoin.sources.FixPriceRateSource;
+import com.generalbytes.batm.server.extensions.extra.futurocoin.sources.yobit.YobitRateSource;
+import com.generalbytes.batm.server.extensions.extra.futurocoin.wallets.futurocoind.FuturocoinRPCWallet;
+import com.generalbytes.batm.server.extensions.watchlist.IWatchList;
+
+import java.math.BigDecimal;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.StringTokenizer;
+
+//import com.generalbytes.batm.server.extensions.extra.dogecoin.sources.chainso.ChainSoRateSource;
+//import com.generalbytes.batm.server.extensions.extra.dogecoin.wallets.blockio.BlockIOWallet;
+
+public class FuturocoinExtension implements IExtension{
```

Comment:
Don't forget to register the extension (together with the implemented sources and wallets) in ```batm-extensions.xml``` file.